samples/client/petstore/powershell_test/src/IO.Swagger/API/PetApi.ps1

```diff
@@ -0,0 +1,164 @@
+function PetApi-addPet {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = , ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
```

Comment:
Template failed to generate `Position` attribute correctly (missing position number after `=`)